provisioner/template.go

```diff
@@ -95,6 +95,7 @@ func renderTemplate(context *templateContext, file string) (string, error) {
 		"azID":                 azID,
 		"azCount":              azCount,
 		"split":                split,
+		"join":                 strings.Join,
```

Comment:
Maybe it makes sense to revert argument order like here https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/621/files#diff-807fefe6626b30fd095cee0c1c6cf81f2219f513b244dd13ba403395786734fdR281-R285 to enable piped join like `{{ $foo | join "," }}`

Reply:
Changed!